docs/GettingStarted.md

```diff
@@ -140,6 +141,10 @@ This is a test application for the sample eBPF extension. This application loads
 and attaches it to the test hook implemented by the sample extension and validates if the eBPF program
 executed as expected.
 
+### xdp_tests.exe
+This application tests various XDP functionalities. This how the tests should be run:
+1. Reflection Test: This tests the XDP_TX functionality. On the system under test, install eBPF binaries, then load reflect_packet.o using netsh, ebpf, add program commands. Then from a remote host, run xdp_tests.exe and in `--remote-ip` parameter pass the IP address of the system under test.
```

Comment:
Replace "then... then..." with a sequence of numbered steps.

Also if I understand right, you can't use just any IP address of the system under test, it has to be the IP address of an Ethernet-like interface.